.travis.yml

```diff
@@ -94,6 +96,14 @@ matrix:
         # Report success since we have overridden default behaviour
         - bash -c "$STATUS" success "Local $NAME testing has passed"
 
+#    - <<: *tools-pytest
+#      env: NAME=tools-py3.5
+#      python: 3.5
+#
+#    - <<: *tools-pytest
+#      env: NAME=tools-py3.6
+#      python: 3.6
```

Comment:
nice!

why not turn it on?

Reply:
Ignore me, I didn't read your message close enough :+1: 

Let's keep this pr open until python 3 support works again. Maybe we should create a branch for python3 work?

Reply:
@geky Hmm, a feature branch might not be that bad of an idea, although I'm not confident it's needed. We actually already have some tools in master that are python3 ready, courtesy of @theotherjimmy.

I'd prefer to not keep PRs open as a progress indicator, mainly because I don't like to see PRs linger :)

@0xc0170 thoughts?

Reply:
A branch will quickly become out of date with master and then be rebase hell, I think.

Reply:
Fair enough

It just seems strange to merge something that is entirely commented out

Reply:
That's fair. The flip side is that devs can enable the functionality to test porting efforts :)